openvm/src/lib.rs

```diff
@@ -205,7 +221,11 @@ pub fn compile_exe(
 ) -> Result<CompiledProgram<F>, Box<dyn std::error::Error>> {
     // Build the ELF with guest options and a target filter.
     // We need these extra Rust flags to get the labels.
-    let guest_opts = GuestOptions::default();
+    let guest_opts = if let Some(features) = guest_features {
+        GuestOptions::default().with_features(features)
+    } else {
+        GuestOptions::default()
+    };
```

Comment:
I think instead of passing `guest_features: Option<Vec<String>>` and repeating this code, we should pass `GuestOption` directly?

Reply:
See my comment above :)